Packs/CiscoUmbrellaReporting/pack_metadata.json

```
@@ -2,7 +2,7 @@
     "name": "Cisco Umbrella Reporting",
     "description": "Use Cisco Umbrella's Reporting to monitor your Umbrella integration and gain a better understanding of your Umbrella usage. Gain insights into request activity and blocked activity, determining which of your identities are generating blocked requests. Reports help build actionable intelligence in addressing security threats including changes in usage trends over time. The Umbrella Reporting v2 API provides visibility into your core network and security activities and Umbrella logs. This integration was integrated and tested with version 2 of Cisco-umbrella-reporting.",
     "support": "xsoar",
-    "currentVersion": "1.1.2",
+    "currentVersion": "1.2.0",
```

This would be only a revision -[ ref ](https://xsoar.pan.dev/docs/documentation/release-notes#examples:~:text=This%20command%20automatically,or%20updating%20documentation.)

```suggestion
    "currentVersion": "1.1.3",
```

---

you're right, my bad

---

Please also update the name of the release notes file accordingly.